### DISCUSSION TITLE
tutorials/intro-dgs/08-query-arguments

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#107] [course feedback]
# tutorials/intro-dgs/08-query-arguments

https://www.apollographql.com/tutorials/intro-dgs/08-query-arguments

**Comment by mkatsman** [comment]
I got this: {
  "data": {
    "playlist": null
  }
}. Perhaps the hard code index is not valid. Just mention it that it can change. Or provide another way of obtaining the id.

**Reply by mkatsman** [reply]
sorry, my fault. Left behind the @DgsQuery annotation.

**Reply by mkatsman** [reply]
no error on the server for the invalid request or anything.

**Reply by mabuyo** [reply]
Hey there @mkatsman - just to clarify, you fixed the issue?

**Reply by mkatsman** [reply]
I did. Thank you. I am a little confused with the fact, that there is no server error, in case the end point is not recognized. In restful AI this would be 404. Anyway to turn some debugging on?
